docs/pages/setup/guides/ssh-key-extensions.mdx

```diff
@@ -0,0 +1,45 @@
+---
+title: Teleport with GitHub.
```

Comment:
I think this title should be more descriptive. There's already [a guide](https://goteleport.com/docs/setup/admin/github-sso/) to using Teleport with GitHub SSO, for example—how is this different from that guide? What is it that users will achieve after completing this guide?